src/asm/fstack.c

```diff
@@ -464,19 +464,19 @@ void fstk_RunForeach(char const *symName, int32_t start, int32_t stop, int32_t s
 	else if (step < 0 && stop < start)
 		count = (start - stop - 1) / -step + 1;
 	else if (step == 0)
-		error("FOREACH cannot have a step value of 0\n");
+		error("FOR cannot have a step value of 0\n");
 
 	if (count == 0)
 		return;
 	if (!newReptContext(reptLineNo, body, size))
 		return;
 
 	contextStack->nbReptIters = count;
-	contextStack->foreachValue = start;
-	contextStack->foreachStep = step;
-	contextStack->foreachName = strdup(symName);
-	if (!contextStack->foreachName)
-		fatalerror("Not enough memory for FOREACH name: %s\n", strerror(errno));
+	contextStack->forValue = start;
+	contextStack->forStep = step;
+	contextStack->forName = strdup(symName);
+	if (!contextStack->forName)
+		fatalerror("Not enough memory for FOR name: %s\n", strerror(errno));
```

Comment:
While we're at this, maybe correct this to "FOR **symbol** name"?

Reply:
Done.